docs/contracts.md

```diff
@@ -4,7 +4,9 @@ title: Contracts
 sidebar_label: Contracts
 ---
 
-The smart contracts that we have currently deployed are the following:
+We have currently deployed or are working on deploying the following smart contracts:
 
-1. Gnosis Safe on the Harmony Blockchain
-1. *TBA*
+1. Gnosis Safe on the Harmony blockchain
+1. Multisig on the Cardano blockchain
```

Comment:
There is no Cardano multisig from what I know. We could say: "Wrapped Cardano Gnosis Safe"

Reply:
```suggestion
1. Wrapped Cardano Gnosis Safe on the Cardano blockchain
```